flash_tutorials/electricity_forecasting/.meta.yml

```diff
@@ -0,0 +1,20 @@
+title: Electricity Price Forecasting with NBeats
+author: Ethan Harris (ethan@pytorchlightning.ai)
+created: 2021-11-23
+updated: 2021-11-23
+license: CC BY-SA
+tags:
+  - Tabular
+  - Forecasting
+description: |
+  This tutorial covers using Lightning Flash and it's integration with PyTorch Forecasting to train an autoregressive
+  model (NBeats) on hourly electricity pricing data. We show how the built-in interpretability tools from PyTorch
+  Forecasting can be used with Flash to plot the trend and daily seasonality in our data discovered by the model. We
+  also cover how features from PyTorch Lightning such as the learning rate finder can be used easily with Flash. As a
+  bonus, we show hat we can resample daily observations from the data to discover weekly trends instead.
+requirements:
+  - pandas==1.1.5
```

Comment:
Reviewers note: This version is being pinned just for now until an issue with more recent pandas versions is resolved in Flash